### PR TITLE
MDS-3109 Update bond_view to include Active record of un-etl-able REL/CON bonds

### PR DESCRIPTION
**Pull request by nayr974**

### CODE REVIEW

File: migrations/sql/R__view_bonds.sql [hunk 1]
```diff
@@ -101,6 +101,35 @@ FROM
                   null as "update_user", 
                   'Yes' AS "is_current_record", 
                   'MMS_FDW' as "source"
-     FROM SES_BOND_STAGING) AS bond_data
+     FROM SES_BOND_STAGING
```
Comment (exzizt) [inline migrations/sql/R__view_bonds.sql:104]: On line 87 of this file we should be setting the closed date.

**Review by exzizt — CHANGES_REQUESTED** [review]
This solution looks correct, nice work. But I noticed that we should be setting the closed_date for the closed records on line 87.

File: migrations/sql/R__view_bonds.sql (hunk 1 above)
Comment (nayr974) [reply migrations/sql/R__view_bonds.sql:104]: Added logic to set closed date :)

**Review by exzizt — APPROVED** [review]
🍏 📗 🟢 💚 🥗 ✔️ ✅

**Pull request merged**